setup.py

```diff
@@ -20,7 +20,7 @@
     author='Boyan Soubachov',
     author_email='boyanvs@gmail.com',
     url='http://pypi.python.org/pypi/tastyworks/',
-    version='4.0.0',
+    version='4.0.2',
```

Comment:
This is way more than a minor bug-fix :) Could you bump it up to 4.1.0 as it adds new functionality.